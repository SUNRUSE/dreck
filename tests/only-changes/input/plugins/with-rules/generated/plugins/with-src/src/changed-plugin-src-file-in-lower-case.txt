changed plugin src file in subdirectory (ephemeral).
