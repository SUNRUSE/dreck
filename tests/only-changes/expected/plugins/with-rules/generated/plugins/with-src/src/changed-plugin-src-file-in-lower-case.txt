changed plugin src file (src).
